Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/BooleanSetting.java

```diff
@@ -173,6 +173,10 @@ public enum BooleanSetting implements AbstractBooleanSetting
   GFX_WIDESCREEN_HACK(Settings.FILE_GFX, Settings.SECTION_GFX_SETTINGS, "wideScreenHack", false),
   GFX_CROP(Settings.FILE_GFX, Settings.SECTION_GFX_SETTINGS, "Crop", false),
   GFX_SHOW_FPS(Settings.FILE_GFX, Settings.SECTION_GFX_SETTINGS, "ShowFPS", false),
+  GFX_SHOW_VPS(Settings.FILE_GFX, Settings.SECTION_DEBUG, "ShowVPS", false),
+  GFX_SHOW_SPEED(Settings.FILE_GFX, Settings.SECTION_DEBUG, "ShowSpeed", false),
+  GFX_SHOW_FRAME_TIMES(Settings.FILE_GFX, Settings.SECTION_DEBUG, "ShowFrame_times", false),
+  GFX_SHOW_SPEED_COLORS(Settings.FILE_GFX, Settings.SECTION_DEBUG, "ShowSpeedColors", true),
```

Comment:
These all should be in `Settings.SECTION_GFX_SETTINGS` and the frame time setting key needs to be changed to "ShowFrameTimes"